src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/ExplicitConversion.cs

```diff
@@ -141,12 +141,7 @@ public bool Bind()
                         return false; // Can't convert to a method group or anon method.
                     case TypeKind.TK_NullType:
                         return false;  // Can never convert TO the null type.
-                    case TypeKind.TK_TypeParameterType:
-                        if (bindExplicitConversionToTypeVar())
-                        {
-                            return true;
-                        }
-                        break;
+
```

Comment:
nit: remove empty line?

Reply:
Do we have a guideline on spacing between cases? I normally have a space before all but the first in a switch, and have been doing that most of the time I touch on cases as part of another change.

Reply:
It is about consistency. Either we have a single empty line before all but the first in a switch, or none at all.